src/commands/help.js

```diff
@@ -0,0 +1,99 @@
+//
+// Written by J10a1n15.
+// See LICENSE for copyright and license notices.
+//
+
```

Comment:
One line instead of two /j

Reply:
what im confused

Reply:
There's only one line between the header comment and the code

Reply:
![image](https://github.com/PartlySaneStudios/sanity/assets/45315647/089fccdb-f55c-4df1-b4f8-a8ba897927cc)
should it not be only one?

Reply:
Thought it was 2

Reply:
idk, is there a style for it?

Reply:
Idk I just like 2